Assignment4/Assignment4.js

```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+
+function readAppendWrite (readFile, appendFile, writeFile) {
+  fs.readFile (readFile, function (err1, data1) {
```

Comment:
`fs.readFile (readFile, function (err1, data1)` should be `fs.readFile(readFile, function (err1, data1)`.
Problem Type: Spacing